modules/ROOT/pages/manage-message-flows.adoc

```diff
@@ -31,19 +31,23 @@ You can also undeploy or delete message flows entirely if your partner relations
 * xref:endpoint-as2-receive.adoc[AS2 Endpoint Settings]
 
 * xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
-
-. If you want to upload a different DataWeave map, click *Import* from the Map section to choose a different DataWeave map and click *Upload mapping*.
-
++
+. If you want to replace your existing DataWeave map, click *Import* from the Map section to choose a different DataWeave map and click *Upload mapping*.
```

Comment:
Does import automatically replace the existing map?

Reply:
Currently, it does. I'm not sure if users will ever have an option to upload multiple maps.